### PR TITLE
Replaced references of Freenode with OFTC

### PR DESCRIPTION
**Pull request by panos**

### CODE REVIEW

**Comment by panos** [comment]
This is an early change and updates are to follow.

File: content/community/irc/_index.html [hunk 1]
```diff
@@ -10,6 +10,8 @@ <h3>Chat channels</h3>
 <p>Haiku has chat channels in various languages, allowing realtime discussion
 with other Haiku developers and users.</p>
 
+<p><b>NOTE:</b> We are currently in the process of transitioning most of our channels from the Freedom IRC network to the OFTC IRC network.</p>
```
Comment (pulkomandy) [inline content/community/irc/_index.html:13]: I think we have a dedicated css class to make a nice alert box (with icon and everything).

File: content/community/irc/_index.html [hunk 2]
```diff
@@ -19,21 +21,21 @@ <h4>Connection instructions</h4>
...
 <a name="haiku"></a>
-<h3><a href="irc://chat.freenode.net/haiku">Main IRC channel</a> (English)</h3>
+<h3><a href="irc://irc.oftc.net/haiku">Main IRC channel</a> (English)</h3>
 <p>General discussion (on and off-topic) for the Haiku community</p>
 <ul>
-<li><strong>Network:</strong> Freenode</li>
-<li><strong>Channel:</strong> <a href="irc://chat.freenode.net/haiku">#haiku</a></li>
+<li><strong>Network:</strong> OFTC</li>
+<li><strong>Channel:</strong> <a href="irc://irc.oftc.net/haiku">#haiku</a></li>
 <li><a href="https://freenode.logbot.info/haiku">Channel logs</a></li>
```
Comment (pulkomandy) [inline content/community/irc/_index.html:29]: this needs to be updated too (once we have the new logbot set up)

File: content/development/faq.html [hunk 3]
```diff
@@ -62,7 +62,9 @@ <h3>Do you have a mailing list for developers?</h3>
 <a name="irc"></a>
 <h3>Do you have an IRC channel?</h3>
 
-<p>There are multiple <a href="/community/irc">IRC Channels</a>. Currently, all of the listed ones are on the irc.freenode.net network. [<a href="#top">top</a>]</p>
```
Comment (pulkomandy) [inline content/development/faq.html:65]: maybe this question should be reworded in a more general way, as some channels (haiku-es and haiku-ru) are in fact on Telegram

**Pull request merged**